### PR TITLE
"No OQL" warning on plots, mutations, co expression tabs

### PR DESCRIPTION
**Pull request by adamabeshouse**
The screenshots are a bit confusing in this regard, but to clarify: the message is the same size on each tab (its the same jsp injected), but it is not placed the same, i.e. with the same padding. I can change that, I wasn't sure what the cleanest way to handle that would be though (help me @alisman ?)

image

image

image

### CODE REVIEW

File: portal/src/main/webapp/WEB-INF/jsp/mutation_details.jsp [hunk 1]
```diff
@@ -29,8 +29,12 @@
  - You should have received a copy of the GNU Affero General Public License
  - along with this program.  If not, see <http://www.gnu.org/licenses/>.
 --%>
+
 <div class='section cbioportal-frontend' id='mutation_details'>
-    <img src='images/ajax-loader.gif' alt='loading'/>
+    <jsp:include page="global/no_oql_warning.jsp" flush="true" />
+    <div id="mutation_details_inner">
```
Comment (adamabeshouse) [inline portal/src/main/webapp/WEB-INF/jsp/mutation_details.jsp]: @alisman I dont think I messed anything up here, but please pay extra attention

**Comment by jjgao** [comment]
added @schultzn to get his approval.

**Comment by jjgao** [comment]
@adamabeshouse is it possible to display the message when OQL is used?

**Comment by adamabeshouse** [comment]
@jjgao you mean "this tab reflects the OQL specification from your query" ?

**Comment by jjgao** [comment]
@adamabeshouse: I mean if the user just queries genes, it may not be necessary to show the message. Is it possible to display the message only when something like "EGFR: MUT" is queried. @tmazor thoughts?

**Comment by tmazor** [comment]
I agree - the message should only be displayed if the user has used OQL in their query. If the query is just for gene names with no OQL, then there's no need to display the warning.

**Comment by alisman** [comment]
@adamabeshouse lets discuss if we should maybe try to do this new codebase instead.  also would be nice to use bootstrap alert for consistency with other alerts.  lets talk.  sorry i'm late to the discussoin here.

**Comment by adamabeshouse** [comment]
@alisman good idea

**Comment by tmazor** [comment]
Hi all - any update on when this may get deployed? @adamabeshouse @jjgao @alisman

**Comment by tmazor** [comment]
hey guys - any update on this? @adamabeshouse @alisman

**Comment by adamabeshouse** [comment]
since we're going to merge plots in the next release, let's do it at the same time, all those tabs will be in react now

**Pull request closed without merging**